gazelle/parse.py

```diff
@@ -38,25 +42,44 @@ def parse_comments(content):
     return comments
 
 
-def parse(stdout, filepath):
-    with open(filepath, "r") as file:
+def parse(repo_root, rel_package_path, filename):
+    rel_filepath = os.path.join(rel_package_path, filename)
+    abs_filepath = os.path.join(repo_root, rel_filepath)
+    with open(abs_filepath, "r") as file:
         content = file.read()
-        modules = parse_import_statements(content)
-        comments = parse_comments(content)
+        with concurrent.futures.ThreadPoolExecutor(max_workers=2) as executor:
```

Comment:
where did you come up with the number 2 here? maybe worth a comment

Reply:
Addressed with [9732ff7](https://github.com/bazelbuild/rules_python/pull/630/commits/9732ff794232ec4c13819be5e16b00fc32d77e74).